pkgs/development/python-modules/graphlan/default.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  biopython,
+  matplotlib,
+  scipy,
+}:
+
+buildPythonPackage rec {
```

Comment:
So I had another look, and this looks more like an application?
In which case it goes in pkgs/by-name, using buildPythonApplication.

Reply:
Just checked the dependents listed on GitHub, you seem to be correct. I'll move it there when I get to updating everything.